### PR TITLE
Add per crucible stats

### PR DESCRIPTION
**Pull request by pakaplace**

### CODE REVIEW

File: src/context/web3.tsx [hunk 1]
```diff
@@ -64,28 +68,47 @@ const Web3Provider: React.FC = (props) => {
...
+  // GET USER'S LP DEPOSITS
   const { loading, error, data } = useQuery(GET_UNISWAP_MINTS, {
     variables: { userAddress: address },
     skip: !address, // Must have address to query uniswap LP's
   });
 
+  // GET PAIR HISTORY FOR MINTS
+  const [loadPairs, { loading: loadingPairs, data: pairData }] = useLazyQuery(
+    createPairHistoryQuery(
+      "0xcd6bcca48069f8588780dfa274960f15685aee0e",
```
Comment (nuurdev) [inline src/context/web3.tsx:106]: You can pull the lpToken from config

Comment (nuurcodes) [inline src/context/web3.tsx:106]: You can pull the lpToken address from config

File: src/context/web3.tsx [hunk 2]
```diff
@@ -64,28 +68,47 @@ const Web3Provider: React.FC = (props) => {
...
+      crucibles.length
+        ? crucibles.map((crucible) => crucible.mintTimestamp)
+        : [1615464000]
```
Comment (nuurcodes) [inline src/context/web3.tsx:109]: Seems like an important timestamp, what does it indicate?

**Review by nuurcodes — APPROVED** [review]

File: src/context/web3.tsx (hunk 2 above)
Comment (pakaplace) [reply src/context/web3.tsx:115]: It's a fallback, I'll remove in `crucible-frontend`

File: src/context/web3.tsx (hunk 1 above)
Comment (pakaplace) [reply src/context/web3.tsx]: Done

**Pull request merged**